cdc/processor/pipeline/sorter.go

```diff
@@ -120,7 +120,7 @@ func createSorter(ctx pipeline.NodeContext, tableName string, tableID model.Tabl
 			dbActorID := ssystem.DBActorID(uint64(tableID))
 			compactScheduler := ctx.GlobalVars().SorterSystem.CompactScheduler()
 			levelSorter, err := db.NewSorter(
-				ctx, tableID, startTs, ssystem.DBRouter, dbActorID,
+				ctx, ctx.ChangefeedVars().ID, tableID, startTs, ssystem.DBRouter, dbActorID,
```

Comment:
please do not use `ctx` to pass changefeed information.

Reply:
I will refine this in next PR.